Tribler/Core/Utilities/torrent_utils.py

```diff
@@ -86,16 +87,22 @@ def create_torrent_file(file_path_list, params, torrent_filepath=None):
     if params.get('nodes'):
         for node in params['nodes']:
             torrent.add_node(*node)
-    # HTTP seeding
-    # http://www.bittorrent.org/beps/bep_0017.html
-    if params.get('httpseeds'):
-        torrent.add_http_seed(params['httpseeds'])
 
-    # Web seeding
-    # http://www.bittorrent.org/beps/bep_0019.html
-    if len(file_path_list) == 1:
-        if params.get('urllist', False):
-            torrent.add_url_seed(params['urllist'])
+    from Tribler.Core.Libtorrent.LibtorrentMgr import LibtorrentMgr
+    if LooseVersion(LibtorrentMgr.get_libtorrent_version()) < LooseVersion("1.2.0"):
+        # In libtorrent 1.2.0, adding http/url seeds to a torrent requires a Boost.Python converter for
+        # basic_string_view, which is not available yet.
```

Comment:
oh, this should be added to the binding, right?

Reply:
Yes, but I’m not sure how to do so.

Reply:
would you mind filing a ticket against libtorrent? and maybe help me with something simple to put in my `test.py` to exercise the problem

Reply:
Sure, no problem 👍

Reply:
I just created an issue.

@arvidn Is a run of the tests in `test.py` is included in the libtorrent CI pipeline? If not, I think it would be helpful to prevent regression errors.

Reply:
it is